docs/android/ci.md

```diff
@@ -0,0 +1,111 @@
+---
+title: "Android Continuous Integration and Delivery"
+sidebar_label: "Continuous Integration"
+---
+
+🚧🚧🚧 Under construction 🚧🚧🚧
+
+## Android Continuous Integration and Delivery
+
+This document outlines the Continuous Integration (CI) and Continuous Delivery (CD) processes for the Android project. We use **GitHub Actions** as our CI/CD platform, with multiple workflows configured to ensure code quality, automate builds, and streamline deployments.
+
+## Overview
+
+The main goals of our CI/CD process are:
+
+- ✅ Validate that everything is working as expected.
+- 🚨 Notify relevant people if something breaks.
+- 🚀 Enable fully automated continuous delivery of applications.
+- 🔄 Avoid duplication by extracting common code into reusable local actions under `.github/actions`.
+
+## Versioning
+
+We follow the same versioning convention as the core project, using [CalVer] (Calendar Versioning). This ensures consistency across all releases.
```

Comment:
```suggestion
We follow the same versioning convention as the core project, using [CalVer] (Calendar Versioning). This ensures consistency across all releases.
```

the link looks off :)

Reply:
It is at the bottom at the file to avoid duplicating the URL. We need to be careful to not remove the space with Calendar Versioning